scraper-ngoindia/scraper.py

```diff
@@ -0,0 +1,88 @@
+from bs4 import BeautifulSoup
+import requests
+
+ngo_content_list = []
+
+def get_page_data():
+    # Specify url to scrape from
+    target_url = requests.get("https://ngosindia.com/ngos-of-india/")
+    page_data = BeautifulSoup(target_url.content, "html.parser")
+    state_list = page_data.find_all(attrs={'class': 'textwidget'})[1].find_all('a',href=True)
+    #go through each state (check if states is empty)
+    for state in state_list:
+        #these states do not have NGO information listed on this website
+        if "Delhi" in state or "Daman and Diu" in state or "Haryana" in state:
+            continue
+        state_url = requests.get(state['href'])
+        #check that the page_data is in the correct format for getting the list
+        state_page_data = BeautifulSoup(state_url.content,"html.parser")
+        has_ngo_data = state_page_data.find_all(attrs={'class': 'ngo-layout-cell'})
+        if has_ngo_data:
+            list = has_ngo_data[1].find_all('a',href=True)
+            if list:
+                #page is at ngo-level
+                if "NGOs" not in str(list[0]):
+                    get_ngo_page_fromngos(list)
+                else:
+                    #page is at county-level
+                    get_ngo_page_fromcounty(list)
+        else:
+            continue
```

Comment:
This else block with the continue isn't needed - it's already at the end of the loop and if the if clause isn't taken it'll automatically jump to the next iteration of the loop